extensions/validate-address/functions/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "shipengine-validate-address-functions",
   "scripts": {
+    "prepare": "npm run build",
```

Comment:
Thank you! I was scouring for solutions last night.

Do you have any documentation on this?